src/dlgAboutDialog.cpp

```diff
@@ -94,4 +98,698 @@ dlgAboutDialog::dlgAboutDialog(QWidget* parent) : QDialog(parent)
     }
 
     mudletTitleLabel->setPixmap(QPixmap::fromImage(splashImage));
+    // clang-format off
+    // Do not let code reformatting tool mess this around!
```

Comment:
We don't need a comment for a comment telling the formatter not to format! It only reads one.

Reply:
The second is my justification to (human) coders/hackers as to why the first is there (to the formatter) - I am trying to keep it as readable as possible and the layout is my best effort to do so - letting clang-format "clean" it up when it does not correspond to the general "style" is not going to leave me feeling warm and fluffy!

Reply:
We know what it means though, so it's unnecessary prose we have to read. We don't explain every single detail of the C++ standard either or how Qt works :stuck_out_tongue: